gabc/details.html

```diff
@@ -165,7 +165,7 @@ <h3 id="determination">Determination of the neume according to the sequence of n
 <p>To any sequence of notes and spaces corresponds one and only one written form. The only exception is the sequence of three or more ascending notes. To determine the written form to use in this case, we must introduce a character to prevent  linking between two notes. This character is <code>!</code> .  It allows us to have a complete equivalence between our notation and the written form.  But be aware that this sign indicates a space of zero width: it is the only "space" that can be found within a glyph.</p>
```

Comment:
Didn't we have some sort of fix which affected this thing?  In any event, we'll need to update this diagram and the accompanying text.